scripts/get_aap_plaintext_docs.sh

```diff
@@ -9,19 +9,28 @@ fi
 AAP_VERSION=$1
 LIGHTSPEED_LATEST=lightspeed-latest
 AAP_CLOUDS_LATEST=aap-clouds-latest
-BASE_DIR=aap-product-docs-plaintext
+AAP_DOCS_BASE_DIR=aap-product-docs-plaintext
+CHANGELOGS_FILE=additional_docs/components_versions.txt
 
 set -eou pipefail
 
 trap "rm -rf aap-docs" EXIT
 
-if [ ! -d ${BASE_DIR} ]; then
-    mkdir ${BASE_DIR}
+if [ ! -d ${AAP_DOCS_BASE_DIR} ]; then
+    mkdir ${AAP_DOCS_BASE_DIR}
 fi
 
+# Changelog's
+if [ ! ${CHANGELOGS_FILE} ]; then
+    touch ${CHANGELOGS_FILE}
+fi
+echo -e "# Ansible Components Versions\n\nHere are the most recent updates for Ansible components:\n" > $CHANGELOGS_FILE
+./scripts/generate_changelog.sh "Ansible Core (ansible-core)" "https://api.github.com/repos/ansible/ansible/releases/latest" $CHANGELOGS_FILE
+./scripts/generate_changelog.sh "Ansible Rulebook (ansible-rulebook)" "https://api.github.com/repos/ansible/ansible-rulebook/releases/latest" $CHANGELOGS_FILE
+
```

Comment:
do we want to do the same for all the docs, i.e. dynamically generate this at build time, and get rid of that text file from the git?

Reply:
I was expecting to be able to delete the `component_versions.txt` that @TamiTakamiya included manually, but honestly, I think we still can't, I prefer to talk to him before. This is because I couldn't found any reference, in the build tasks, to a call to the script that generates the plain docs text files. 
I understand this is still a manual process. As I don't have many days remaining and I have another task to complete, I was not getting deeper. 
For now doesn't hurt, once @TamiTakamiya is back we can double check I guess :)
Thx!

Reply:
BTW for all docs I don't think it applies, this is just calling the GH API to get release information, so it helps to have all component versions up to date.
But docs are docs... :) No need for a GH endpoint, we just need to process and digest, as @TamiTakamiya did, and keep improving all this cycle.

Reply:
Agreed when @TamiTakamiya is back we can re-iterate on the current logic

Reply:
agree, as much we can automate, much better!

Reply:
@romartin (cc: @justjais @jameswnl) Thanks for working on this. The code looked good.

I was also thinking of the idea of dynamically generating the `component_versions.txt`  during my PTO. I thought we may be able to utilize [Jinja2 templates](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_templating.html), which is used in Ansible, i.e. rename `components_versions.txt` to `components_versions.j2` and define it something like:
```
# Ansible Components Versions

Here are the most recent updates for Ansible components:

- Ansible Core (ansible-core) Version {{ latest_ansible_component_versions['ansible-core'] }} Released: {{ latest_ansible_latest_release_dates['ansible-core'] }} 
- Ansible Rulebook (ansible-rulebook) {{ latest_ansible_component_versions['ansible-rulebook'] }} Released: {{ latest_ansible_latest_release_dates['ansible-rulebook'] }}
```
then run a separate playbook to retrieve those variables. 

Though currently this is just an idea, combining chatbot (not only in pre-process and but also in runtime though [function calling](https://platform.openai.com/docs/guides/function-calling)) with Ansible seems to have lots of possibilities.  What do you think?